inst/metadata/metadata.csv

```diff
@@ -1,28 +1,28 @@
-data_type,plot_type,indicator,value_column,error_low_column,error_high_column,indicator_column,indicator_value,name,scale,accuracy,format
-survey,choropleth,prevalence,est,,,indicator,prev,t_(HIV_PREVALENCE),1,,0.00%
-survey,choropleth,art_coverage,est,,,indicator,artcov,t_(ART_COVERAGE),1,,0.00%
-programme,choropleth,current_art,current_art,,,,,t_(ART_NUMBER_ATTENDING),1,,0
-survey,choropleth,recent,est,,,indicator,recent,t_(PROPORTION_RECENTLY_INFECTED),100,100,0
-survey,choropleth,vls,est,,,indicator,vls,t_(VLS),1,,0.0%
-anc,choropleth,prevalence,prevalence,,,,,t_(HIV_PREVALENCE),1,,0.00%
-anc,choropleth,art_coverage,art_coverage,,,,,t_(ART_COVERAGE),1,,0.00%
-output,choropleth,prevalence,mean,lower,upper,indicator_id,2,t_(HIV_PREVALENCE),1,,0.00%
-output,choropleth,art_coverage,mean,lower,upper,indicator_id,4,t_(ART_COVERAGE),1,,0.00%
-output,choropleth,current_art,mean,lower,upper,indicator_id,5,t_(ART_NUMBER_RESIDENTS),1,,0
-output,choropleth,receiving_art,mean,lower,upper,indicator_id,6,t_(ART_NUMBER_ATTENDING),1,,0
-output,choropleth,population,mean,lower,upper,indicator_id,1,t_(POPULATION),1,1000,0
-output,choropleth,plhiv,mean,lower,upper,indicator_id,3,t_(PLHIV),1,100,0
-output,choropleth,incidence,mean,lower,upper,indicator_id,7,t_(INCIDENCE),1000,,0.00
-output,choropleth,new_infections,mean,lower,upper,indicator_id,8,t_(NEW_INFECTIONS),100,100,0
-output,choropleth,anc_prevalence,mean,lower,upper,indicator_id,9,t_(ANC_HIV_PREVALENCE),1,,0.00%
-output,choropleth,anc_art_coverage,mean,lower,upper,indicator_id,10,t_(ANC_PRIOR_ART_COVERAGE),1,,0.0%
-output,barchart,prevalence,mean,lower,upper,indicator_id,2,t_(HIV_PREVALENCE),1,,0.00%
-output,barchart,art_coverage,mean,lower,upper,indicator_id,4,t_(ART_COVERAGE),1,,0.00%
-output,barchart,current_art,mean,lower,upper,indicator_id,5,t_(ART_NUMBER_RESIDENTS),1,,0
-output,barchart,receiving_art,mean,lower,upper,indicator_id,6,t_(ART_NUMBER_ATTENDING),1,,0
-output,barchart,population,mean,lower,upper,indicator_id,1,t_(POPULATION),1,1000,0
-output,barchart,plhiv,mean,lower,upper,indicator_id,3,t_(PLHIV),1,100,0
-output,barchart,incidence,mean,lower,upper,indicator_id,7,t_(INCIDENCE),1000,,0.00
-output,barchart,new_infections,mean,lower,upper,indicator_id,8,t_(NEW_INFECTIONS),100,100,0
-output,barchart,anc_prevalence,mean,lower,upper,indicator_id,9,t_(ANC_HIV_PREVALENCE),1,,0.00%
-output,barchart,anc_art_coverage,mean,lower,upper,indicator_id,10,t_(ANC_PRIOR_ART_COVERAGE),1,,0.0%
+data_type,plot_type,indicator,value_column,error_low_column,error_high_column,indicator_column,indicator_value,name,scale,accuracy,format
+survey,choropleth,prevalence,est,,,indicator,prevalence,t_(HIV_PREVALENCE),1,,0.00%
+survey,choropleth,art_coverage,est,,,indicator,art_coverage,t_(ART_COVERAGE),1,,0.00%
+programme,choropleth,art_current,art_current,,,,,t_(ART_NUMBER_ATTENDING),1,,0
+survey,choropleth,recent_infected,est,,,indicator,recent_infected,t_(PROPORTION_RECENTLY_INFECTED),100,100,0

```

Comment:
The `value_column` needs to be updated here too - in the test data this has been renamed to `estimate`.

Reply:
Thanks very much for catching these. Addressed in 8a5c7f3.